templates/profile/info.yml.twig

```diff
@@ -0,0 +1,26 @@
+name: '{{ profile }}'
+type: 'profile'
+description: '{{ description }}'
+core: '8.x'
+distribution:
```

Comment:
Correction, TooManyDistributionsException never made it in. I thought it did as part of the "Write install_profile value to configuration and only to settings.php if it is writeable (#2156401)". 

So I'/m actually not sure what happens here. Certainly, the lines between profile and distribution are further blurred by this.